python/aghast/interface.py

```diff
@@ -5033,8 +5033,8 @@ def _fromflatbuffers(cls, fb):
 
     def _toflatbuffers(self, builder):
         sumw = self.sumw._toflatbuffers(builder)
-        sumw2 = None if self.sumw2 is None else self.sumw2._toflatbuffers(builder)
-        unweighted = None if self.unweighted is None else self.unweighted._toflatbuffers(builder)
+        sumw2 = self.sumw2._toflatbuffers(builder) if self.sumw2 is not None else None
+        unweighted = self.unweighted._toflatbuffers(builder) if self.unweighted is not None else None
```

Comment:
The corrections (same following) look right. I was expecting something like this.